master/internal/core.go

```diff
@@ -1084,12 +1084,12 @@ func (m *Master) Run(ctx context.Context, gRPCLogInitDone chan struct{}) error {
 		return err
 	}
 
-	command.RegisterAPIHandler(
-		m.system,
-		m.echo,
-		m.db,
-		m.rm,
-	)
+	// Wait for all NTSC services to initialize.
+	command.SetDefaultCmdService(m.db, m.rm)
```

Comment:
I would usually call this init. I would expect Set to take a CommandService, not construct one for the caller.